apps/ledger-live-desktop/src/renderer/components/SyncOnboarding/Manual/SyncOnboardingCompanion.tsx

```diff
@@ -479,12 +481,25 @@ const SyncOnboardingCompanion: React.FC<SyncOnboardingCompanionProps> = ({
     };
   }, [device, allowedError, handleDesyncTimerRunsOut, desyncTimeout]);
 
+  const recoverServices = useFeature("protectServicesDesktop");
+  const recoverStoreId = recoverServices?.params?.protectId ?? "";
+
+  const [recoverOnboardingStatus, setRecoverOnboardingStatus] = useState<boolean>();
+
+  const getRecoverOnboardingStatus = useCallback(() => {
+    const userId = getUserId();
```

Comment:
I'm not sure it's the right way to get the user id, you can use the `getOrCreateUser` method however

Reply:
We need to use [the same function](https://github.com/LedgerHQ/ledger-live/blob/develop/apps/ledger-live-desktop/src/helpers/user.ts#L4-L16) Wallet API is using this is to ensure the storage variable name is correct regardless of the live app that requests it as it references Wallet API's `userId`

Reply:
Ok go it 👍